real-main/app/models/user/manager.py

```diff
@@ -464,6 +465,23 @@ def on_user_delete_delete_cognito(self, user_id, old_item):
             # TODO: catch 404 error & log warning
             self.cognito_client.delete_identity_pool_entry(user_id)
 
+    def find_users(self, caller_user, emails=None, phones=None):
+        emails = set(emails) if emails else []
+        phones = set(phones) if phones else []
+
+        user_ids_from_emails = self.email_dynamo.batch_get_user_ids(emails)
+        user_ids_from_phones = self.phone_number_dynamo.batch_get_user_ids(phones)
+        user_ids = set(user_ids_from_emails + user_ids_from_phones) or []
```

Comment:
```suggestion
        user_ids = list(set(user_ids_from_emails + user_ids_from_phones)) or []
```

This is the cause of the bug with the return value from `app.handlers.appsync.handlers.find_users`. It was a typing issue... returning a `set` vs a `list`.

Reply:
resolved